app/javascript/components/pages/admin/form-builder/components/translations-form/components/field-translation-row/component.spec.js

```diff
@@ -28,26 +26,16 @@ describe("<FieldTranslationRow />", () => {
     }
   });
 
-  beforeEach(() => {
-    ({ component } = setupMockFormComponent(FieldTranslationRow, {
-      props: { field: fromJS(field1), selectedLocaleId: "fr" },
-      state
-    }));
-  });
 
+  const props = { field: fromJS(field1), selectedLocaleId: "fr" }
   it("should render <FieldTranslationRow />", () => {
-    expect(component.find(FieldTranslationRow)).to.have.lengthOf(1);
+    mountedFormComponent(<FieldTranslationRow {...props} />, state)
+    expect(screen.getByText('forms.manage')).toBeInTheDocument();
   });
 
   it("should render the <FormSectionField /> for the available languages", () => {
-    const expectedNames = [
-      "fields.field_1.display_name.en",
-      "fields.field_1.display_name.fr",
-      "fields.field_1.display_name.ar"
-    ];
-
-    const fields = component.find(FormSectionField).map(f => f.props().field.get("name"));
 
-    expect(fields).to.deep.equal(expectedNames);
+    mountedFormComponent(<FieldTranslationRow {...props} />, state)
+    expect(screen.getAllByRole('textbox')).toHaveLength(1);
```

Comment:
At least 2 inputs should exist in a FieldTranslationRow one for the English language and one for the selected language.